server/routerlicious/packages/services-core/src/documentManager.ts

```diff
@@ -0,0 +1,10 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { IDocument } from "./document";
+
+export interface IDocumentManager {
```

Comment:
nit: IDocumentReader ? Does not like we are going to add other document managing operations to this interface.

Reply:
Where is `IDocumentManager` used?

Reply:
It will be used in FRS override. expose here because I feel this is a basic class (That internal http request to alfred) that can be used in OSS too

Reply:
@arikt-ms I will leave it as it for now. Just incase for the future, like abstract the direct documentCollection deletion to this service. It will be easier to extend without breaking changes.